Resources/Textures/_Goobstation/Clothing/OuterClothing/Hardsuits/dreadnought.rsi/meta.json

```
@@ -1,7 +1,8 @@
 {
   "version": 1,
   "license": "CC-BY-SA-3.0",
-  "copyright": "Sprited by @b11jj for goobstation",
+  "copyright": "Sprite made by Gtheglorious based on the sprite made by emisse for ss14, vox state made by Flareguy for SS14, Resprited by Sunrise, monkey state made by MercerBray for Ss14.",
```

Biome (2.4.15)

[error] The key copyright was already declared.

(lint/suspicious/noDuplicateObjectKeys)